### ISSUE TITLE
MapBox StyleSource

### ISSUE THREAD

**Issue by lcorallo** [ApuliaCreativeHub/E-Culture-Tool#22]
### ** Pre-announcement **
Let's consider the following dependency for MapBox: 10.2.0
Change in the build.gradle file:
`implementation" com.mapbox.maps: android: 10.1.0 "` in `implementation" com.mapbox.maps: android: 10.2.0 "`

Relying entirely on the official documentation (it does not matter if Android or JS, the functional structure is the same) and on the Javadoc documentation of the classes.

### ** Implementation **
The functionality to be implemented will be present in the Fragment `com.apuliacreativehub.eculturetool.ui.places.MapFragment.java`.

In particular we need to change line 96 (approximately) `map.getMapboxMap (). LoadStyleUri (Style.MAPBOX_STREETS);` to:
> `map.getMapboxMap (). loadStyleUri (Style.MAPBOX_STREETS, new Style.OnStyleLoaded () {`
>>> `@ Override`
>>> `public void onStyleLoaded (@NonNull Style style) {`
>>> `// Content Here`
> `}`
`}`

Basically from v10 the MapboxBox and the MapView are synchronized. Previously, a special function was used to synchronize them. (map.getAsync () something like this).

The style must then be loaded into the function.

### map.addStyleSource (id: String, properties. Values): <String, None>
The attention has gone particularly on this function which allows to set the id, and its relative style.
Properties can be manageable through the Values ​​class. The function returns an empty string if everything is ok. Otherwise an error message.

The documentation can be found directly on the ide, a useful link is the following:
- https://docs.mapbox.com/mapbox-gl-js/style-spec/sources (note that this is an explanation for JS, the properties structure is the same.
